project-auth/src/main/scala/io/renku/projectauth/Role.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2023 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.projectauth
+
+import cats.Order
+import cats.data.NonEmptyList
+import io.circe.{Decoder, Encoder}
+
+sealed trait Role extends Ordered[Role] {
```

Comment:
I'm wondering if `Role` shouldn't go to Renku tiny types as a common concept everywhere. I think we already have it implemented in some way in some other place(s). We could maybe reuse the new `Role` and use it everywhere.

Reply:
Ah good point, thanks. I need to look where we already use a role